task_geo/data_sources/covid/cds/cds_formatter.py

```diff
@@ -12,12 +12,20 @@ def cds_formatter(df):
 
     assert df[pd.isnull(df.country)].empty
 
-    df = df.rename(columns={'growthFactor': 'growth_factor'})
+    df = df.rename(columns={
+        'growthFactor': 'growth_factor',
+        'state': 'region',
+        'county': 'sub_region'
+    })
     df = df.reindex(columns=[
-        'city', 'county', 'state', 'country', 'population', 'lat', 'long',
```

Comment:
Date should come directly after sub_region

Reply:
I agree that `population`should not be returned.
On the other hand, date should come after all geopolitical columns, it's specified in the [data model documentation.](https://coronawhy.github.io/task-geo/data_model.html#time-related-data).

Reply:
Well then we need to change the data model doc. The primary keys should come at the beginning of the datast.